app/Modules/Core/Components/AbstractBaseControl.php

```diff
@@ -10,49 +10,51 @@
 
 abstract class AbstractBaseControl extends Control
 {
-	/**
+    /**
 	 * @var \Kdyby\Translation\Translator
 	 */
 	protected $translator;
 
 
-	public function __construct(Translator $translator)
-	{
-		$this->translator = $translator;
-	}
+    public function __construct(Translator $translator)
```

Comment:
Good catch! `PhpCsFixer\Fixer\Whitespace\IndentationTypeFixer` will help with that